### PR TITLE
fix(condo):INFRA-296 logging pipeline adjustments

### PR DESCRIPTION
**Pull request by sitozzz**
- [x] pass request_id from app ingress to app headers
- [x] remove http logger at all
- [x] add log at the earliest moment of gql request (requestdidstart maybe)

### CODE REVIEW

File: packages/keystone/KSv5v6/v5/prepareKeystone.js [hunk 1]
```diff
@@ -203,6 +203,9 @@ function prepareKeystone ({ onConnect, extendKeystoneConfig, extendExpressApp, s
                     // nosemgrep: javascript.express.security.audit.remote-property-injection.remote-property-injection
                     req['id'] = req.headers[requestIdHeaderName.toLowerCase()] = reqId
                     res.setHeader(requestIdHeaderName, reqId)
+                    if (!IGNORE_PATH.has(req.url)) {
+                        httpLogger.info({ msg: 'request received', req, res, reqId })
```
[the remark this answers is not in the document]
Comment (sitozzz) [reply packages/keystone/KSv5v6/v5/prepareKeystone.js:207]: In that context you actually don't have an response body. I'll give you an example of what logging here

[the remark this answers is not in the document]
Comment (sitozzz) [reply packages/keystone/KSv5v6/v5/prepareKeystone.js:207]: image

**Review by SavelevMatthew — APPROVED** [review]

**Comment by sitozzz** [comment]
I need to complete several tasks that we found while researching the current configuration

File: packages/keystone/logging/GraphQLLoggerApp.js [hunk 2]
```diff
@@ -39,32 +39,50 @@ function renderExecutableDefinitionNode (node) {
     return ''
 }
 
+/**
+ * Get current time in nanoseconds and return diff between
+ * @param time {bigint} start time of the operation
+ * @returns {number} Operation elapsed time in milliseconds
+ */
+function endTimer (time) {
+    function roundTo (decimalPlaces, numberToRound) {
```
Comment (SavelevMatthew) [inline packages/keystone/logging/GraphQLLoggerApp.js:48]: We can use native JS for this :)
image

File: packages/keystone/logging/GraphQLLoggerApp.js [hunk 3]
```diff
@@ -74,17 +92,16 @@ class GraphQLLoggerPlugin {
             async didEncounterErrors (requestContext) {
                 const logData = getGraphQLReqLoggerContext(requestContext)
                 const errors = get(requestContext, 'errors', [])
-                const responseTime = Number(process.hrtime.bigint() - startTime) / 1000000
```
Comment (SavelevMatthew) [inline packages/keystone/logging/GraphQLLoggerApp.js:77]: This one is calculated somewhere else?

Comment (sitozzz) [reply packages/keystone/logging/GraphQLLoggerApp.js:77]: Yup, total responseTime will be shown at the final event of the apollo lifecycle. didEncounterErrors called before willSendResponse, so IMO we don't need to calculate time for error throwing process

File: packages/keystone/logging/GraphQLLoggerApp.js [hunk 4]
```diff
@@ -39,32 +39,50 @@ function renderExecutableDefinitionNode (node) {
     return ''
 }
 
+/**
+ * Get current time in nanoseconds and return diff between
+ * @param time {bigint} start time of the operation
+ * @returns {number} Operation elapsed time in milliseconds
+ */
+function endTimer (time) {
```
Comment (SavelevMatthew) [inline packages/keystone/logging/GraphQLLoggerApp.js:47]: Maybe `function timeFrom(...)`

**Review by SavelevMatthew — APPROVED** [review]

File: packages/keystone/logging/GraphQLLoggerApp.js (hunk 2 above)
Comment (sitozzz) [reply packages/keystone/logging/GraphQLLoggerApp.js:48]: Good suggestion, I'll refactor it with `toFixed`

File: packages/keystone/logging/GraphQLLoggerApp.js (hunk 4 above)
Comment (sitozzz) [reply packages/keystone/logging/GraphQLLoggerApp.js]: Done

**Pull request merged**